html.editor.lib/test/unit/data/testfiles/syntaxtree/issues145821.html

```diff
@@ -1,3 +1,19 @@
+ <!--
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
This seems to be a specific test data. Does the test work with the header? I would rather exclude it from "rat" task.